cg/meta/orders/api.py

```diff
@@ -171,6 +171,10 @@ def submit_mip_balsamic(self, data: dict) -> dict:
         """Submit a batch of samples for sequencing and analysis."""
         return self.submit_family_samples(data)
```

Comment:
```suggestion
        return self.submit_case_samples(data)
```

Reply:
this suggestion can be applied to most of the functions above this line as well ;)

Reply:
fixed